.ci/travis_script.sh

```diff
@@ -5,7 +5,9 @@ set -e -x
 if [[ -n ${TOXENV} ]]; then
   tox -e ${TOXENV}
 elif [[ "${BIGCHAINDB_DATABASE_BACKEND}" == mongodb ]]; then
-  pytest -v --database-backend=mongodb --cov=bigchaindb
+  pytest -sv --database-backend=mongodb --cov=bigchaindb
```

Comment:
I added `-s` for debugging Travis builds. I would prefer to leave it as is to provide as much debug info for failed tests as possible. Cool?